### ISSUE TITLE
CADD score not displayed

### ISSUE THREAD

**Issue by 47KW** [Clinical-Genomics/scout#2146]
Hi, CADD score for missense etc are not displayed AND scores for other variants are given with many decimals.

https://scout.scilifelab.se/cust002/F0036238/variants

**Comment by dnil** [comment]
Thanks; we noted and fixed part of this last week (#2134 #2135) but the missing ones on that case look conspicuous! We'll have another look in the VCF! 👍

**Comment by dnil** [comment]
And with fixed I mean we fixed it on the dev branch - releasing to production any day now. 😅

**Comment by dnil** [comment]
Yes, kind of looks like only the difficult variants, that previously had no CADD get one via run, whereas the ones that used to have one by lookup didn't get one? This is at any rate already on the VCF, so not a Scout issue. I'll refer it back to the MIP architects! Thank you for reporting!
